scylla/src/utils/test_utils.rs

```diff
@@ -52,3 +54,35 @@ pub(crate) async fn supports_feature(session: &Session, feature: &str) -> bool {
         .split(',')
         .any(|f| f == feature)
 }
+
+/*
+  Creates a generic session builder based on conditional compilation configuration
+  For SessionBuilder of DefaultMode type, adds localhost to known hosts, as all of the tests
+  connect to localhost.
+*/
```

Comment:
Nit: we usually use `// line comments` for item descriptions.

Reply:
Done.